kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/dynamo/lapidary.js

```diff
@@ -18,7 +20,8 @@ onEvent('recipes', (event) => {
             { input: '#forge:gems/amber', energy: 160000 }
         ]
     };
+
     data.recipes.forEach((recipe) => {
-        event.recipes.thermal.lapidary_fuel(recipe.input).energy(recipe.energy * multiplier);
+        event.recipes.thermal.lapidary_fuel(recipe.input).energy(recipe.energy * multiplier).id(`${id_prefix}${recipe.input.replace('#forge:gems/', '')}`);
```

Comment:
what's going on here?

Reply:
this is one of the files where it took little effort to write a proper recipe id instead of assigning md5 to all of them, it basically uses the input tags above it (hard to see if you don't open the [entire](https://github.com/EnigmaticaModpacks/Enigmatica6/blob/9c8c44a14e43f73d3ca231235c132238503dabb6/kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/thermal/dynamo/lapidary.js) file) and removes `#forge:gems/` from it.